shell.nix

```diff
@@ -10,7 +10,9 @@ let
 
   pythonEnv = (pkgs.python3.withPackages (ps: [
     ps.pytest
-    ps.pyelftools
+    (ps.pyelftools.overrideAttrs (old: {
+      doInstallCheck = false;
```

Comment:
I also pushed the fix upstream:

https://github.com/NixOS/nixpkgs/pull/118593/files

Reply:
The updated nixpkgs version now has the fix.